CMSgov/onemac
# Oy2 22002 - date submitted relabel

Story: https://qmacbis.atlassian.net/browse/OY2-22002
Endpoint: See github-actions bot comment

### Details

Update labels from Date Submitted to Initial Submission Date

### Changes

- updated label in both package list and detail view for all package types

### Test Plan

1. Login as any user with view access (state submitter, CMS)
2. Navigate to the package list page and verify the column is now "Initial Submission Date"
3. Navigate to each package type and verify the details page now has "Initial Submission Date" if it once contained "Date Submitted". Note that not every detail page shows this field.
